proto/zetachain/zetacore/observer/params.proto

```diff
@@ -30,6 +30,9 @@ message ChainParams {
   ];
   bool is_supported = 16;
   string gateway_address = 17;
+
+  // Advanced confirmation parameters for chain to support fast observation
+  ConfirmationParams confirmation_params = 18 [ (gogoproto.nullable) = false ];
```

Comment:
Please indicate what the behavior can/should be for the `confirmation_count` this field is set. Will it be ignored?

Also set `uint64 confirmation_count = 1 [ deprecated = true ];`

Reply:
Not going to deprecate it.

Reply:
Marked as deprecated: https://github.com/zeta-chain/node/commit/58928a67b2b02c722a021a31ea77ebc39b7eb3b0